### PR TITLE
Adding support for Druid post aggregations

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: panoramix/static/widgets/viz_helloworld.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+px.registerViz('helloworld', function(slice) {
```
Comment (coleslaw) [inline panoramix/static/widgets/viz_helloworld.js]: What is this? unrelated code?

File: panoramix/views.py [hunk 2]
```diff
@@ -114,6 +114,19 @@ class DruidMetricInlineView(CompactCRUDMixin, PanoramixModelView):
 appbuilder.add_view_no_menu(DruidMetricInlineView)
 
 
+class DruidPostAggregatorInlineView(CompactCRUDMixin, PanoramixModelView):
```
Comment (coleslaw) [inline panoramix/views.py]: Is this a new table? why not a column from the existing table?

**Comment by coleslaw** [comment]
Left some comments, but LGTM :+1:

**Comment by mistercrunch** [comment]
Feature isn't complete yet, that was a first draft.

**Comment by rmessner** [comment]
I'm really interested in this feature.
How can I help to make it a feature ready to merge.

**Comment by sid88in** [comment]
@mistercrunch any update on this?

**Comment by mistercrunch** [comment]
Opting for https://github.com/airbnb/caravel/pull/418 instead of this PR
